tests/test-cases/Group23-VIC-Machine-Service/23-03-VCH-Create.robot

```diff
@@ -54,10 +61,36 @@ Inspect VCH Config ${name}
 Get VCH ${name}
     Get Path Under Target    vch
     ${id}=    Run    echo '${OUTPUT}' | jq -r '.vchs[] | select(.name=="${name}").id'
+    Set Test Variable    ${id}
 
     Get Path Under Target    vch/${id}
 
 
+Pull Busy Box
+    [Arguments]    ${docker_params}    ${expected_rc}    ${expected_output}
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker -H ${docker_params} pull busybox
```

Comment:
should be `${busybox}`

Reply:
I believe we actually want `busybox` in this case, since `${busybox}` comes from harbor (https://github.com/vmware/vic/pull/7727#discussion_r181495422).

Reply:
Added on master: #8089